skopt/gp_opt.py

```diff
@@ -91,16 +91,17 @@ def gp_minimize(func, dimensions, base_estimator=None, acq="LCB", xi=0.01,
         lbfgs is run for 10 iterations optimizing the acquisition function
         over the Gaussian prior.
 
-    * `maxiter` [int, default=1000]:
-        Number of iterations to find the minimum. Note that `n_start`
-        iterations are effectively discounted, such that total number of
-        function evaluations is at most `maxiter`.
+    * `n_calls` [int, default=100]:
+        Number of calls to `func`.
+        If `n_random_starts` > 0, `n_calls - n_random_starts`
+        additional evaluations of `func` are made that are guided
+        by the `base_estimator`.
 
     * `n_points` [int, default=500]:
         Number of points to sample to determine the next "best" point.
         Useless if search is set to `"lbfgs"`.
 
-    * `n_start` [int, default=10]:
+    * `n_random_starts` [int, default=10]:
         Number of random initialization points.
```

Comment:
Maybe we should update this as well for something like "Number of evaluations of `func` with random initialization points."